pom.xml

```diff
@@ -68,6 +68,12 @@
         <hadoop.version>3.2.0</hadoop.version>
       </properties>
     </profile>
+    <profile>
+      <id>dataproc-2.0</id>
+      <properties>
+        <hadoop.version>3.2.2</hadoop.version>
+      </properties>
+    </profile>
```

Comment:
Is this the BKM verified on GCP by @HongW2019 ?

Reply:
Yes, I confirmed with @HongW2019 for GCP support yesterday and all TPC-DS queries can be passed by using this change.

@HongW2019 closed this issue yesterday, but we actually haven't supported it in the source code.
This PR is to fix it.

Reply:
BTW, besides Hadoop version support, there is no other place to be changed for GCP support.